example/reinforcement-learning/a3c/a3c.py

```diff
@@ -153,8 +153,11 @@ def train():
                 module.forward(batch, is_train=True)
 
                 pi = module.get_outputs()[1]
-                h = args.beta*(mx.nd.log(pi+1e-6)+1)
-                module.backward([mx.nd.array(adv), h])
+                h = -args.beta*(mx.nd.log(pi+1e-7)*pi)
+                out_acts = np.amax(pi.asnumpy(), 1)
```

Comment:
i refer to some tensorflow implentation. Actually, if we add value loss term, the result may be better, but i think this  beyond the original paper.